source/simulator/checkpoint_restart.cc

```diff
@@ -97,6 +97,7 @@ namespace aspect
       oa << parameters.names_of_compositional_fields;
       oa << parameters.normalized_fields;
       oa << parameters.mesh_deformation_enabled;
+      oa << parameters.walltime_from_resume;
```

Comment:
This is the wrong place. This function is here to save user parameters to make sure no crucial option got changed by the user in the prm

Reply:
I'm not sure where I am supposed to save "walltime_from_resume" then so that this value is read in again on a restart?

Reply:
Take a look at https://github.com/geodynamics/aspect/blob/589ef4bed0f3f2887ea287182da665648baaa1ec/source/simulator/checkpoint_restart.cc#L599

Reply:
Great, thanks!